releases/release-1.25/release-notes/release-notes-draft.md

```diff
@@ -1,19 +1,83 @@
+
+### Promoted core CSI Migration to Stable
+
+[CSI Migration](https://kubernetes.io/blog/2021/12/10/storage-in-tree-to-csi-migration-status-update/#quick-recap-what-is-csi-migration-and-why-migrate) is an ongoing effort that SIG Storage has been working on for a few releases. The goal is to move in-tree volume plugins to out-of-tree CSI drivers and eventually remove the in-tree volume plugins. The [core CSI Migration](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration) feature moved to GA. CSI Migration for GCE PD and AWS EBS also moved to GA. CSI Migration for vSphere remains in beta (but is on by default). CSI Migration for Portworx moved to Beta (but is off-by-default).
+
+
+### Promoted CSI Ephemeral Volume to Stable
+
+The [CSI Ephemeral Volume](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/596-csi-inline-volumes) feature allows CSI volumes to be specified directly in the pod specification for ephemeral use cases. They can be used to inject arbitrary states, such as configuration, secrets, identity, variables or similar information, directly inside pods using a mounted volume. This was initially introduced in 1.15 as an alpha feature, and it moved to GA. This feature is used by some CSI drivers such as the [secret-store CSI driver](https://github.com/kubernetes-sigs/secrets-store-csi-driver).
+
+### Promoted CRD Validation Expression Language to Beta
+
+[CRD Validation Expression Language](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2876-crd-validation-expression-language/README.mdv) is promoted to beta, which makes it possible to declare how custom resources are validated using the [Common Expression Language (CEL)](https://github.com/google/cel-spec). Please see the [validation rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) guide.
+
+### Promoted Server Side Unknown Field Validation to Beta
+
+Promoted the `ServerSideFieldValidation` feature gate to beta (on by default). This allows optionally triggering schema validation on the API server that errors when unknown fields are detected. This allows the removal of client-side validation from kubectl while maintaining the same core functionality of erroring out on requests that contain unknown or invalid fields.
+
+
+###  Introduced KMS v2
+
+Introduce KMS v2alpha1 API to add performance, rotation, and observability improvements. Encrypt data at rest (ie Kubernetes `Secrets`) with DEK using AES-GCM instead of AES-CBC for kms data encryption. No user action is required. Reads with AES-GCM and AES-CBC will continue to be allowed. See the guide [Using a KMS provider for data encryption](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/) for more information.
+
+
+### Removed AKS and GKE authentication from kubectl
```

Comment:
Is this supposed to move to deprecation and removal?

Reply:
Hi @cici37 
Could rephrase your question?

The capability is now removed from v1.25, users need to use an external to kubectl plugin/tool to authenticate with AKS and GKE.

Reply:
Sorry I might put it into the wrong place. Since Major Theme would be part of the release blog and we have a section in release blog specifically for Deprecation and Removal so I am wondering if it should be part of that section. No worry. We could rearrange in release blog if needed. Thank you for putting it together!

Reply:
Thank you @cici37 for the clarification
Yes, this is a removal that should be included in release blog under deprecations and removals